apiserver/facades/controller/firewaller/firewaller_base_test.go

```diff
@@ -231,82 +230,6 @@ func (s *firewallerBaseSuite) testWatchModelMachines(
 	wc.AssertNoChange()
 }
 
-const (
```

Comment:
Note, this test was moved from the `firewaller_base_test` to the `firewaller_test` suite because there were already mocks implement there. There was conditional logic for a condition that was never set to true so this was also removed.